crates/iceberg-catalog/src/service/authz/mod.rs

```diff
@@ -153,6 +153,9 @@ pub enum NamespaceParent {
 
 #[async_trait::async_trait]
 /// Interface to provide AuthZ functions to the catalog.
+/// The provided `Actor` arguments of the all methods except `check_actor`
+/// are assumed to be valid. Please ensure to call `check_actor` before, preferably
+/// during Authentication.
```

Comment:
maybe specify what you mean by valid in this context?

Reply:
All other occurances are in tests - no other places

Reply:
I meant for future reference it could make sense to expand a bit on validity, i.e. it has been checked that the currently authenticated user is allowed to use this actor?